tests/integration/multiple_submissions/conftest.py

```diff
@@ -12,7 +12,12 @@
 @pytest.fixture
 def ingest_api():
     api = IngestApi()
-    api.set_token(f"Bearer {os.getenv('INGEST_TOKEN')}")
+    # TODO could also use a token generator here using the GCP service account,
```

Comment:
Nice idea!
I would move this token generation functionality to the ingest client.